docs/source/examples/supplychain/supplychain_transaction_family.rst

```diff
@@ -43,61 +43,121 @@ Agent
 Agents represent owners or custodian of goods in the system. All agents must
 be registered in the system before they can interact with records.
 
-.. literalinclude:: ../../../../families/supplychain/protos/agent.proto
```

Comment:
You mean embed the entirety of every proto-file including the Intel header? You could probably cut out the header without worrying about the offsets too much since that probably isn't changing.

BUT, the Agent Transactions sections would have to be completely rewritten somehow, as every example is a piece the same file. I did not want to include any rewrites in the scope of this PR, so this is one of the reasons I decided against `literalinclude`. I suppose I could front-load the section with the entirety of the `payload.proto` file, and delete the code examples from all the subsections. That seems not great though.